app/main.py

```diff
@@ -1,8 +1,31 @@
 class Person:
-    # write your code here
-    pass
+    people: dict = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    spouse_types: set[str] = {"wife", "husband"}
+    for person in people:
+        if person["name"] in Person.people:
+            continue
+        current_person = Person(person["name"], person["age"])
+        Person.people[person["name"]] = current_person
```

Comment:
why do u need this ? the person will be added to dict in init method

Reply:
Agree, that action is redundant